ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -442,18 +443,30 @@ mlag_port_channel_interfaces:
 
 - `{{ mlag_interfaces }}`
```

Comment:
@ClausHolbechArista - consider making it more clear that this is a list. i.e
```suggestion
- `{{ mlag_interfaces[] }}`
```

Reply:
```suggestion
- `{{ mlag_interfaces }}` (list of strings)
```